src/options/set_language.cpp

```diff
@@ -37,37 +36,37 @@ SetLanguage::Scope::~Scope(){
   SetLanguage::setLanguage(d_out, d_oldLanguage);
 }
 
-
-SetLanguage::SetLanguage(OutputLanguage l)
-  : d_language(l)
-{}
+SetLanguage::SetLanguage(Language l) : d_language(l) {}
 
 void SetLanguage::applyLanguage(std::ostream& out) {
   // (offset by one to detect whether default has been set yet)
   out.iword(s_iosIndex) = int(d_language) + 1;
 }
 
-OutputLanguage SetLanguage::getLanguage(std::ostream& out) {
+Language SetLanguage::getLanguage(std::ostream& out)
+{
   long& l = out.iword(s_iosIndex);
   if(l == 0) {
     // set the default language on this ostream
     // (offset by one to detect whether default has been set yet)
     if(not Options::isCurrentNull()) {
-      l = options::outputLanguage() + 1;
+      l = static_cast<long>(options::outputLanguage()) + 1;
```

Comment:
Can we use a fixed width type here?

Reply:
Not really: `std::ios_base::iword()` returns a `long`, and I'd rather stay with this type.